website/docs/cloud-docs/api-docs/explorer.mdx

```diff
@@ -0,0 +1,460 @@
+  Types](#view-types) for a list of fields available for each type.
+- **FIELD_OPERATOR**: The operator to use when filtering. Must be supported by
+the field type being filtered. See [Filter Operators](#filter-operators) for the
+available operators and their supported field types.
+- **FIELD_VALUE_INDEX**: Must be `0`. This sub-parameter is reserved for future use.
+- **FILTER_VALUE**: The filter value used by the filter during the query.
+
+Once the desired filter strings have been added to a request URI, they should be
+percent-encoded along with the rest of the query string parameters.
+
+#### Sample Filter Strings
+
+**View Type: `workspace`**
+
+_Show workspaces that contain `test` in their name:_
+
+```
+filter[0][workspace_name][contains][0]=test
+```
+
+_After encoding:_
+```
+filter%5B0%5D%5Bworkspace_name%5D%5Bcontains%5D%5B0%5D=test
+```
+
+**View Type: `modules`**
+
+_Show modules that contain `aws` in their name and version string equal to `1.1`_
+
+```
+filter[0][name][contains][0]=aws&filter[0][version][is][0]=1.1
+```
+
+_After encoding:_
+```
+filter%5B0%5D%5Bname%5D%5Bcontains%5D%5B0%5D=aws&filter%5B0%5D%5Bversion%5D%5Bis%5D%5B0%5D=1.1
+```
+
+#### Filter Operators
+
+| Operator           | Supported Field Types         | Description |
+| ------------------ | ----------------------------- | ----------- |
+| `is`               | `boolean`, `number`, `string` | Filters data to rows where the field value is equivalent to the filter value. |
+| `is_not`           | `number`, `string`            | Filters data to rows where the field value is different from the filter value. |
+| `contains`         | `string`                      | Filters data to rows where the field value contains the filter value as a sub-string. |
+| `does not contain` | `string`                      | Filters data to rows where the field value does not contain the filter value as a sub-string. |
+| `is_empty`         | `boolean`, `number`, `string` | Filters data to rows where the field does not contain a value. |
+| `is_not_empty`     | `boolean`, `number`, `string` | Filters data to rows where the field contains any value. |
+| `gt`               | `number`                      | Filters data to rows where the field value is greater than the filter value. |
+| `lt`               | `number`                      | Filters data to rows where the field value is less than the filter value. |
+| `gteq`             | `number`                      | Filters data to rows where the field value is greater than or equal to the filter value. |
+| `lteq`             | `number`                      | Filters data to rows where the field value is less than or equal to the filter value. |
+| `is_before`        | `datetime`                    | Filters data to rows where the field value is earlier than the filter value. ISO 8601 formatted dates are recommended for filter values. If no time zone offset is provided, the filter value will be interpreted as UTC. |
```

Comment:
Are other options than ISO 8601 formatted dates supported? As best I can tell, we're pretty prescriptive that dates and times are all in ISO 8601 format, but just want to make sure we're being exact here.

Reply:
I think that rails is able to parse timestamps with slight differences, but I see no reason not to use more exact language.